### PR TITLE
DOC: First commit of Windows installation instructions

### PR DESCRIPTION
**Pull request by mohammedkhalfan**
First contribution. Not sure if this is sufficient. Feedback welcome. Also, hope I did this pull request thing correctly.

### CODE REVIEW

**Comment by tibkiss** [comment]
Thanks for this contribution, much appreciated!

Question regarding page-structure: How do you imagine presenting this page?
Currently the tutorial has MAC specific instructions, whereas this document focuses on Windows dependencies only.

**Comment by mohammedkhalfan** [comment]
On the existing tutorial page, the first part says "Assuming you have Python 2.7 and virtualenv installed, you can install zipline-live using pip:". Here, we can add a line saying "If you're using Windows, see [this page](#) for installation instructions". 

Only the windows specific instructions should be separated. Everything else that is the same like TWS config, sample algo, etc. should be documented only in one place.

**Comment by tibkiss** [comment]
Makes sense. Could you please include the tutorial change (back-reference to windows) part of this PR? Thanks!

File: tutorial.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: On premise live trading with zipline
 # zipline-live with Interactive Brokers TWS
 ## Install
 Assuming you have [Python 2.7](https://www.python.org/) and [virtualenv](https://virtualenv.pypa.io) installed,
-you can install zipline-live using `pip`:
+you can install zipline-live using `pip`. If you're using Windows, see [this page](/windows.md) for installation instructions.
```
Comment (tibkiss) [inline tutorial.md]: please use (/windows) to be consistent with the rest of the page

File: windows.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+# zipline-live installaion on Windows
+
+This tutorial assumes you have a clean Windows installation with Administrator rights. This tutorial has been implemented using Windows 10 on [Microsoft Azure Cloud](https://azure.microsoft.com).
+ 
+## Install Anaconda
+Download and install [Anaconda](https://www.anaconda.com/download/) (Python 2.7).
+
+## Install Microsoft Visual C++ Compiler for Python 2.7
+Download and install from https://www.microsoft.com/en-us/download/details.aspx?id=44266
```
Comment (tibkiss) [inline windows.md]: The link is not clickable, I'd suggest using:
`[here](https://www.microsoft.com/en-us/download/details.aspx?id=44266)`

File: windows.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+## Install ta-lib (optional)
+Open Anaconda Shell as Adminstrator and install talib using `conda`:
+```
+conda install -c quantopian ta-lib
+```
+
+## Download and install IB TWS
+Download, install, and configure IB TWS as shown here http://www.zipline-live.io/tutorial
```
Comment (tibkiss) [inline windows.md]: link is not clickable, i'd suggest using 
[here](/tutorial)

**Comment by tibkiss** [comment]
+1, super thanks for putting this together!

**Pull request merged**